2.feature_extraction/scripts/1.cp_analysis.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Perform segmentation and feature extraction using CellProfiler
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pathlib
+import pprint
+
+import sys
+
+sys.path.append("../utils")
+import cp_parallel
+
+
+# ## Set paths and variables
+
+# In[2]:
+
+
+# set the run type for the parallelization
```

Comment:
Run type is a parameter I created in the `cp_parallel` function that will provide a name to the log file so that you know what type of CellProfiler pipeline was ran, since there are many types. It is a way to provide documentation to the log file so that if I gets moved, you still know the context behind that pipeline it is logging.

The run type for this case is "analysis", which can sound confusing but in the world of CellProfiler, it just means a segmentation and feature extraction pipeline.